modules/openapi-generator/src/main/resources/Java/libraries/webclient/ApiClient.mustache

```diff
@@ -149,20 +165,43 @@ public class ApiClient{{#jsr310}} extends JavaTimeFormatter{{/jsr310}} {
     }
 
     /**
-    * Build the WebClient used to make HTTP requests.
+    * Build the WebClientBuilder used to make WebClient.
     * @return WebClient
     */
-    public static WebClient buildWebClient(ObjectMapper mapper) {
+    public static WebClient.Builder buildWebClientBuilder(ObjectMapper mapper) {
         ExchangeStrategies strategies = ExchangeStrategies
             .builder()
             .codecs(clientDefaultCodecsConfigurer -> {
                 clientDefaultCodecsConfigurer.defaultCodecs().jackson2JsonEncoder(new Jackson2JsonEncoder(mapper, MediaType.APPLICATION_JSON));
                 clientDefaultCodecsConfigurer.defaultCodecs().jackson2JsonDecoder(new Jackson2JsonDecoder(mapper, MediaType.APPLICATION_JSON));
             }).build();
-        WebClient.Builder webClient = WebClient.builder().exchangeStrategies(strategies);
-        return webClient.build();
+        WebClient.Builder webClientBuilder = WebClient.builder().exchangeStrategies(strategies);
+        return webClientBuilder;
+    }
+
+    /**
+     * Build the WebClientBuilder used to make WebClient.
+     * @return WebClient
+     */
+    public static WebClient.Builder buildWebClientBuilder() {
+        return buildWebClientBuilder(createDefaultObjectMapper(null));
+    }
+
+    /**
+     * Build the WebClient used to make HTTP requests.
+     * @return WebClient
+     */
+    public static WebClient buildWebClient(ObjectMapper mapper) {
```

Comment:
@UnleashSpirit thanks for the PR. Can you please add `@param` for `mapper` to the docstring above?

Reply:
done